spec/models/refinery/jobs/job_application_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require 'helper/generator_helper'
+
+module Refinery
+  module Jobs
+    describe JobApplication, type: :model do
+
+      let(:job_application) {FactoryGirl.create(:job_application)}
+      let(:job) {FactoryGirl.create(:job)}
+
+      describe 'validations' do
+        it 'should be functional' do
+          # Negative tests
+          expect(FactoryGirl.build(:job_application, name: nil)).not_to be_valid
+          expect(FactoryGirl.build(:job_application, phone: nil)).not_to be_valid
+          expect(FactoryGirl.build(:job_application, email: nil)).not_to be_valid
+          expect(FactoryGirl.build(:job_application, email: generate_string)).not_to be_valid
+          expect(FactoryGirl.build(:job_application, email: "#{generate_string}@")).not_to be_valid
+          expect(FactoryGirl.build(:job_application, email: "#{generate_string}@{generate_string}.")).not_to be_valid
+          expect(FactoryGirl.build(:job_application, cover_letter: nil)).not_to be_valid
+          expect(FactoryGirl.build(:job_application, resume: nil)).not_to be_valid
+
+          # Positive test
+          expect(FactoryGirl.build(:job_application,
+            { name: generate_string,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.